gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -809,17 +844,18 @@ message TripDescriptor {
     // A trip that existed in the schedule but was removed.
     CANCELED = 3;
 
-    // Should not be used - for backwards-compatibility only.
-    REPLACEMENT = 5 [deprecated = true];
+    // A trip that replaces an existing trip in the schedule.
```

Comment:
```suggestion
    // A trip that replaces an existing trip in the schedule. REPLACEMENT is equivalent to using DELETED for the existing trip and NEW for the replacement trip.
    // NOTE: This field is still experimental, and subject to change. It may be formally adopted in the future.
```
Since this is being undeprecated, I suggest marking it as experimental.

Reply:
After some thoughts I have decided not to add the sentence of "REPLACEMENT is equivalent to using DELETED for the existing trip and NEW for the replacement trip."

DELETED then NEW requires the use of a different trip id but REPLACEMENT is specifically using the same trip id.